components/ScreenReaderOnly/ScreenReaderOnly.module.css

```diff
@@ -1,10 +1,3 @@
 .ScreenReaderOnly {
-  border: 0;
   clip: rect(0, 0, 0, 0);
```

Comment:
how to convert this attribute any suggestions?

Reply:
you'll love this answer: https://tailwindcss.com/docs/screen-readers

just remove all of the CSS and use `sr-only`

Reply:
cool

Reply:
done